assets/javascripts/discourse/models/filter-rule.js.es6

```diff
@@ -8,8 +8,10 @@ export default RestModel.extend({
   category: function() {
     var id = this.get('category_id');
 
-    if (id === "0")
+    if (id === 0 || id === "0")
```

Comment:
If you run a `parseInt` on `id` you wouldn't have to check for string and scalar versions of both